spaces/S000052/properties/P000204.md

```diff
@@ -0,0 +1,9 @@
+---
+space: S000052
+property: P000204
+value: false
+---
+
+Note that for any $a,b,c,d \in X$, the closures of $U_a(b)$ and $U_c(d)$ intersect in infinitely many points.
```

Comment:
```suggestion
The closures of any two base elements $U_a(b)$ and $U_c(d)$ intersect in infinitely many points.
```
preferable, because saying "for any a,b,c,d in X" creates difficulties: without further restrictions, the open $U_a(b)$ does not make sense.